crates/provider/src/evm/rpc/proofs_provider.rs

```diff
@@ -25,14 +23,34 @@ pub struct FetchedAccountProof {
     pub account_proof: Vec<String>,
 }
 
+/// Fetched storage and account proof and it's value
 #[derive(Debug, Clone)]
-pub struct FetchedStorageProof {
+pub struct FetchedStorageAccountProof {
     pub block_number: u64,
+    pub encoded_account: String,
     pub account_proof: Vec<String>,
     pub storage_value: String,
     pub storage_proof: Vec<String>,
 }
 
+#[derive(Debug, Clone)]
+pub struct FetchedTransactionProof {
```

Comment:
can alloy types be used here? coz we call standard rpc methods sh maybe there are nice types for it

Reply:
same